primitives/src/communities.rs

```diff
@@ -376,10 +413,11 @@ pub mod consts {
 mod tests {
 	use crate::{
 		bs58_verify::Bs58Error,
-		common::IpfsValidationError,
+		common::{FromStr as CrateFromStr, IpfsValidationError},
 		communities::{
-			validate_demurrage, validate_nominal_income, CommunityIdentifier, CommunityMetadata,
-			CommunityMetadataError, Degree, Demurrage, Location, NominalIncome, RangeError,
+			validate_demurrage, validate_nominal_income, BoundedCommunityMetadata,
```

Comment:
I would vote for not changing the `CommunityMetadata` name. Does anything speak against that?

Reply:
we need both for testing the migration, but i guess we could go for `CommunityMetadata` and `UnboundedCommunityMetadata`. what do you think?

Reply:
I think this is better 👍

Reply:
Please do the same for the pallet string